study.md

```diff
@@ -50,15 +50,17 @@ In your own words, define what the responsibilities of the controller layer are
 in Rails.
 
 ```md
-<!-- your answer here -->
+The controller takes the data from the model and puts it in the view. You only manipulate data in the controllers.
```

Comment:
We won't be using Rails's views so for our purposes the controllers will render responses to the client.